packages/layouts-react/CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Changelog
+
+## 0.1.2 (2022-05-26)
+
+### What's new
+
+- **Added `PageLayout` component.** It has `PageLayout.Header`, `PageLayout.SideNavigation` and `PageLayout.Content` sub-components that place content in the correct places.
```

Comment:
Do we need to add `ToolsArea` and `TitleArea` too?

Reply:
Good catch, addded